config/locales/display/display-pt.yml

```diff
@@ -45,6 +45,7 @@ pt:
     multi_page:
       categories: Categorias
       describe: Criar metadados
+      pages_need_completion: Páginas que precisam ser completadas
```

Comment:
I dont' speak Portugese, but the other examples are "para ???ar" or "para ???er".  We should look at that here.